check/configuration/supported/readme.md

```diff
@@ -0,0 +1,85 @@
+| Textproto       | [txtpbfmt](https://github.com/protocolbuffers/txtpbfmt/)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| TOML            | [taplo](https://github.com/tamasfe/taplo#readme)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| Typescript      | [deno](https://deno.land/manual), [eslint](https://github.com/eslint/eslint#readme), [prettier](https://github.com/prettier/prettier#readme), [rome](https://github.com/rome/tools#readme), [semgrep](https://github.com/returntocorp/semgrep#readme)                                                                                                                                                                                                                                                                                                                                                |
+| YAML            | [prettier](https://github.com/prettier/prettier#readme), [semgrep](https://github.com/returntocorp/semgrep#readme), [yamllint](https://github.com/adrienverge/yamllint#readme)                                                                                                                                                                                                                                                                                                                                                                                                                       |
+
+1. Support for Detekt is under active development; see our [docs](https://docs.trunk.io/docs/check-supported-linters#detekt) for more details.
+2. [Module inspection](https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/module-inspection.md), [deep checking](https://github.com/terraform-linters/tflint-ruleset-aws/blob/master/docs/deep\_checking.md), and setting variables are not currently supported.
+
+### Linter-specific Configuration
+
+
+Some linters require a bit more modification to properly set up. View page for the particular linter for more info.
```

Comment:
I agree with Eli. I don't think this setup is very ergonomic with 2 tables.

These technology + linter pairings are incorrect.